projects/hslayers/src/common/endpoints/endpoint.interface.ts

```diff
@@ -15,6 +15,7 @@ export interface HsEndpoint {
   type: string;
   title: string;
   url: string;
+  id: string;
```

Comment:
Not sure I like this mandatory attribute.

Reply:
I just thought using Id would be a good way of referencing the endpoint, rather then adding it to each layer over and over.